streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java

```diff
@@ -65,224 +64,205 @@
      * KeyValueStore<String, Long> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<String, Long>keyValueStore());
      * String key = "some-key";
      * Long aggForKey = localStore.get(key); // key must be local (application state is shared over all running Kafka Streams instances)
      * }</pre>
-     * For non-local keys, a custom RPC mechanism must be implemented using {@link
-     * KafkaStreams#allMetadata()} to query the value of the key on a parallel running instance of
-     * your Kafka Streams application.
+     * For non-local keys, a custom RPC mechanism must be implemented using {@link KafkaStreams#allMetadata()} to query
+     * the value of the key on a parallel running instance of your Kafka Streams application.
      * <p>
-     * For failure and recovery the store will be backed by an internal changelog topic that will be
-     * created in Kafka.
-     * Therefore, the store name defined by the Materialized instance must be a
-     * valid Kafka topic name and cannot contain characters other than ASCII alphanumerics, '.', '_'
-     * and '-'.
-     * The changelog topic will be named "${applicationId}-${storeName}-changelog", where
-     * "applicationId" is user-specified in {@link StreamsConfig} via parameter {@link
-     * StreamsConfig#APPLICATION_ID_CONFIG APPLICATION_ID_CONFIG}, "storeName" is the provide store
-     * name defined in {@code Materialized}, and "-changelog" is a fixed suffix.
+     * For failure and recovery the store will be backed by an internal changelog topic that will be created in Kafka.
+     * Therefore, the store name defined by the Materialized instance must be a valid Kafka topic name and cannot
+     * contain characters other than ASCII alphanumerics, '.', '_' and '-'.
+     * The changelog topic will be named "${applicationId}-${storeName}-changelog", where "applicationId" is
+     * user-specified in {@link StreamsConfig} via parameter
+     * {@link StreamsConfig#APPLICATION_ID_CONFIG APPLICATION_ID_CONFIG}, "storeName" is a generated value, and
+     * "-changelog" is a fixed suffix.
      * <p>
      * You can retrieve all generated internal topic names via {@link Topology#describe()}.
      *
      * @param initializer  an {@link Initializer} that computes an initial intermediate aggregation
      *                     result. Cannot be {@code null}.
-     * @param materialized an instance of {@link Materialized} used to materialize a state store.
-     *                     Cannot be {@code null}.
      * @return a {@link KTable} that contains "update" records with unmodified keys, and values that
      * represent the latest (rolling) aggregate for each key
      */
-    KTable<K, VOut> aggregate(final Initializer<VOut> initializer,
-                              final Materialized<K, VOut, KeyValueStore<Bytes, byte[]>> materialized);
+    KTable<K, VOut> aggregate(final Initializer<VOut> initializer);
```

Comment:
I reordered the method from "few parameter" to "more parameters" to make it easier to navigate within the file.